daemon/cmd/daemon_main.go

```diff
@@ -1461,6 +1464,13 @@ func runDaemon() {
 	// Start periodical arping to refresh neighbor table
 	if d.datapath.Node().NodeNeighDiscoveryEnabled() {
 		d.nodeDiscovery.Manager.StartNeighborRefresh(d.datapath.Node())
+	} else {
+		// clean up all arp PERM entries that might have previously set by
+		// a Cilium instance
+		if option.Config.GCNeighEntries {
```

Comment:
Looks like this should be a no-op on an L2-less environment (https://github.com/cilium/cilium/blob/master/pkg/datapath/linux/node.go#L1334).

Reply:
That was added in master / 1.10 it doesn't exist in 1.8 branch